<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>primaryFUSEwindow</class>
 <widget class="QMainWindow" name="primaryFUSEwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowTabbedDocks</set>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="mountPoint_input">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>351</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>240</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pipeline Selection</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="pipeline_select">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set up the pipeline for LyveFuse. &lt;/p&gt;&lt;p&gt;Choose streaming or file caching&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>Streaming</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>File Caching</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Progress</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount Point</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output_textEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>731</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="mount_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>100</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mount</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>481</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ALPHA DESIGN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="unmount_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Unmount</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="debugLogging_action"/>
    <addaction name="debugHealthMonitor_action"/>
    <addaction name="debugTesting_action"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="advancedSettings_action"/>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset_Default_Settings">
   <property name="text">
    <string>Reset Default Settings</string>
   </property>
  </action>
  <action name="advancedSettings_action">
   <property name="text">
    <string>Advanced</string>
   </property>
  </action>
  <action name="debugLogging_action">
   <property name="text">
    <string>Logging</string>
   </property>
  </action>
  <action name="debugHealthMonitor_action">
   <property name="text">
    <string>Health Monitor</string>
   </property>
  </action>
  <action name="debugTesting_action">
   <property name="text">
    <string>Testing</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
